azafea/event_processors/endless/metrics/v3/migrations/0bd56cdf56b9_initial_migration_metrics_v3.py

```diff
@@ -0,0 +1,857 @@
+# type: ignore
+
+"""Initial migration for metrics V3
+
+Revision ID: 0bd56cdf56b9
+Revises:
+Create Date: 2021-05-31 15:29:55.157371
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "0bd56cdf56b9"
+down_revision = None
+branch_labels = ("metrics-3",)
```

Comment:
Do we really want to have a separate branch for that? I think we do, but I’m not sure 🤔.

Reply:
It will be easier to manage the migration files of the two versions and if you want to combine the two branches it will not be very complicated to do